# use the new (0.6) libdiscid URL in binding tests)
# changes in the library are there:
# https://github.com/metabrainz/libdiscid/commit/fbb5f6a9f40181b9acd6b7f42138ff87a8e1d47e
--- a/t/10discid.t	2009-09-23 23:12:46.000000000 +0200
+++ b/t/10discid.t	2013-10-20 21:59:59.040040750 +0200
@@ -28,8 +28,8 @@
 is( $disc->freedb_id, '370fce16');
 is( $disc->last_track_num, 22);
 is( $disc->sectors, 303602);
-is( $disc->submission_url, 'http://mm.musicbrainz.org/bare/cdlookup.html?id=xUp1F2NkfP8s8jaeFn_Av3jNEI4-&tracks=22&toc=1+22+303602+150+9700+25887+39297+53795+63735+77517+94877+107270+123552+135522+148422+161197+174790+192022+205545+218010+228700+239590+255470+266932+288750');
-is( $disc->webservice_url, 'http://mm.musicbrainz.org/ws/1/release?type=xml&discid=xUp1F2NkfP8s8jaeFn_Av3jNEI4-&toc=1+22+303602+150+9700+25887+39297+53795+63735+77517+94877+107270+123552+135522+148422+161197+174790+192022+205545+218010+228700+239590+255470+266932+288750');
+is( $disc->submission_url, 'http://musicbrainz.org/cdtoc/attach?id=xUp1F2NkfP8s8jaeFn_Av3jNEI4-&tracks=22&toc=1+22+303602+150+9700+25887+39297+53795+63735+77517+94877+107270+123552+135522+148422+161197+174790+192022+205545+218010+228700+239590+255470+266932+288750');
+is( $disc->webservice_url, 'http://musicbrainz.org/ws/1/release?type=xml&discid=xUp1F2NkfP8s8jaeFn_Av3jNEI4-&toc=1+22+303602+150+9700+25887+39297+53795+63735+77517+94877+107270+123552+135522+148422+161197+174790+192022+205545+218010+228700+239590+255470+266932+288750');
 
 is( $disc->track_offset(1), 150);
 is( $disc->track_offset(2), 9700);
